assets/src/edit-story/components/library/panes/text/textPane.js

```diff
@@ -107,8 +59,7 @@ function TextPane(props) {
           <MainButton
             onClick={() =>
               insertElement('text', {
-                ...getPresetById('subheading'),
-                content: __('Fill in some text', 'web-stories'),
+                ...DEFAULT_PRESET,
                 width: DEFAULT_ELEMENT_WIDTH,
```

Comment:
Does this constant's value need to be updated? According to the issue description / Figma text elements should be 360 wide.

Reply:
Same as above.